### PR TITLE
[Core] Remove outdated methods from CL

### PR DESCRIPTION
**Pull request by loumalouomega**
**Description**
These methods have been deprecated for years, maybe is a good moment to clean up them

**Changelog**
- Remove outdated methods

I tag API breaker beacuse technically it is, but these methods have not been used even since I started my Thesis 5 years ago

### CODE REVIEW

**Comment by loumalouomega** [comment]
FYI @KratosMultiphysics/structural-mechanics

File: kratos/includes/constitutive_law.h [hunk 1]
```diff
@@ -1340,57 +1247,57 @@ class KRATOS_API(KRATOS_CORE) ConstitutiveLaw : public Flags
      * i.e. 2nd PK stress to Kirchhoff stress
      */
     void ContraVariantPushForward( Matrix& rMatrix,
-				   const Matrix& rF );
```
Comment (AlejandroCornejo) [inline kratos/includes/constitutive_law.h:1343]: TOC things ;)

**Review by AlejandroCornejo — COMMENTED** [review]
ok from my side

File: kratos/includes/constitutive_law.h (hunk 1 above)
Comment (loumalouomega) [reply kratos/includes/constitutive_law.h:1343]: In fact is my IDE...

**Review by philbucher — APPROVED** [review]
thanks

approving under the condition that you checked that those methods are not used any more in the Code

**Comment by loumalouomega** [comment]
> thanks
> 
> approving under the condition that you checked that those methods are not used any more in the Code

Are not used, were used in the legacy structural

**Pull request merged**